ansible/roles/operator-pipeline/templates/openshift/tasks/create-container-image.yml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-container-image
+spec:
+  params:
+    - name: pyxis_cert_path
+      default: ""
+      description: Path to Pyxis certificates. Valid only when internal Pyxis is used.
+    - name: pyxis_key_path
+      default: ""
+      description: Path to Pyxis key. Valid only when internal Pyxis is used.
+    - name: pyxis_url
+      default: https://catalog.redhat.com/api/containers/
+    - name: isv_pid
+      description: isv_pid of the certification project from Red Hat Connect
+    - name: registry
+      description: certification project registry
+    - name: repository_name
+      description: Repository name assigned to certification project from Red Hat Connect
+    - name: container_digest
+      description: imagestream container_digest
+    - name: bundle_version
+      description: Operator bundle version
+    - name: is_latest
+      description: If explicitly set to "true", resulting image will be tagged as "latest"
+  workspaces:
+    - name: pyxis-ssl-credentials
+      description: SSL certificates allowing authentication in Pyxis
+    - name: image-data
+      description: JSON files with data about image retrieved with Skopeo and Podman
+  steps:
+    - name: create-container-image
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
+      env:
+        - name: PYXIS_CERT_PATH
+          value: $(params.pyxis_cert_path)
+        - name: PYXIS_KEY_PATH
+          value: $(params.pyxis_key_path)
+        - name: PYXIS_URL
+          value: $(params.pyxis_url)
+        - name: ISV_PID
+          value: $(params.isv_pid)
+        - name: REGISTRY
+          value: $(params.registry)
+        - name: REPOSITORY
+          value: $(params.repository_name)
+        - name: CONTAINER_DIGEST
+          value: $(params.container_digest)
+        - name: BUNDLE_VERSION
+          value: $(params.bundle_version)
+        - name: IS_LATEST
+          value: $(params.is_latest)
+      script: |
+        #! /usr/bin/env bash
+        set -xe
+        echo "Creating container image"
+
+        IMAGE_SIZE=$(cat $(workspaces.image-data.path)/podman.json | jq '.[0].Size')
```

Comment:
Why extract `IMAGE_SIZE` specifically? Couldn't the `create-container-image` script accept `podman.json` in the same way that it accepts `skopeo.json`?